src/renderers/shared/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMNodeStreamRenderer
+ */
+
+'use strict';
+
+var invariant = require('fbjs/lib/invariant');
+var React = require('react');
+var ReactDOMPartialRenderer = require('ReactDOMPartialRenderer');
+
+var Readable = require('stream').Readable;
```

Comment:
Thanks for the comment; I wasn't aware of `readable-stream`!

I did some reading on the subject, though (in particular, this blog post referenced by the readable-stream README), and it seemed like this is only really a problem for versions of Node <= 0.12. Given that those versions of Node are all deprecated, is there still a good reason to use `readable-stream` that you know of?

(and thanks again for your comment!)